### DISCUSSION TITLE
How to run CasaOs? I restarted my PC and it stopped working

### DISCUSSION THREAD

**Discussion by igorovisk** [IceWhaleTech/CasaOS#1783] [Q&A]
How do I run the service? I restarted my pc and it stopped working.

The first time it started automatically, after the installation. How do I manually start my server? Is there any command? I am using a Linux Xubuntu

**Comment by Mine1115** [comment]
hello there, I got the same problem on Debian and instead of the problem being CasaOS not running, the device I was running it on didn't reconnect to my Wi-Fi when it restarted so that's a good place to start looking for what it is. hope it helps
